tests/core/simulation_generator.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+import random
+from openfisca_core.simulations import Simulation
+from openfisca_core.periods import period as make_period
+
+
+def make_simulation(tax_benefit_system, nb_persons, nb_groups, **kwargs):
+    """
+        Generate a simulation containing nb_persons persons spread in nb_groups groups
+    """
+    simulation = Simulation(tax_benefit_system = tax_benefit_system, **kwargs)
+    simulation.persons.ids = np.arange(nb_persons)
+    simulation.persons.count = nb_persons
+    adults = [0] + sorted(random.sample(xrange(1, nb_persons), nb_groups - 1))
+
+    members_entity_id = np.empty(nb_persons, dtype = int)
+    members_legacy_role = np.empty(nb_persons, dtype = int)
```

Comment:
Would it be helpful to write a reminder of the `legacy_role` indexes ?

Reply:
Added a comment to explain what legacy roles are